pom.xml

```diff
@@ -66,9 +66,9 @@
     <argLine>-Xmx2g -enableassertions ${jacoco-config}</argLine>
   </properties>
 
-  <url>http://qpid.apache.org/proton</url>
+  <url>https://qpid.apache.org/proton</url>
   <scm>
-    <connection>scm:git:http://git-wip-us.apache.org/repos/asf/qpid-protonj2.git</connection>
+    <connection>scm:git:https://git-wip-us.apache.org/repos/asf/qpid-protonj2.git</connection>
```

Comment:
https everywhere, don't see why not

Reply:
Not a spelling error, better added to release tasks JIRA

Reply:
I'll create a Jira for it, PROTON-2419, https://github.com/apache/qpid-protonj2/pull/3